src/components/item/iterable.js

```diff
@@ -5,14 +5,19 @@ const { DragSource, DropTarget, getEmptyImage } = require('../dnd')
 const { compose, map, filter, into } = require('transducers.js')
 const { DND } = require('../../constants')
 const { isMeta } = require('../../keymap')
-const { pure } = require('../util')
+const { bounds } = require('../../dom')
+const { noop, pure } = require('../util')
```

Comment:
Careful, `noop` is defined in `common/util` not `components/util`

The `pure` utility there, by the way, is something that, we may want to replace by `React.memo` nowadays.